c2corg_ui/static/js/utils.js

```diff
@@ -0,0 +1,58 @@
+goog.provide('app.utils');
+
+goog.require('goog.asserts');
+
+
+/**
+ * @param {string} module The document type.
+ * @param {string} documentId The document id.
+ * @param {string} culture Culture.
+ * @return {string} Url.
+ */
+app.utils.buildDocumentUrl = function(module, documentId, culture) {
```

Comment:
Yes and no. "module" is a bit more generic and MVC-related I think.
"type" could also be confusing since we have for instance waypoints types.